### ISSUE TITLE
Bukkit/Spigot plugin pages

### ISSUE THREAD

**Issue by Flying--Dutchman** [cppchriscpp/TravelPortals#34]
Is there any chance to update the Bukkit page of this plugin and to create a Spigot page?
The last update is almost 2 years ago and has a really old version.

Would also offer myself to keep those pages up-to-date, would love to see more players using this plugin.

**Comment by Phoenix616** [comment]
Yeah, I've meant to post an update at some point when I was sure the UUID functionality and other recent changes were stable enough.

Not sure if we want a project page on two sites though as that would require updating both all the time. I personally moved all my early projects over to Spigot (with the exception of ChestShop) but I don't think that's an option with an established plugin. Main issue with Spigot's resource pages would be that only one user can have access to them :S

**Comment by Flying--Dutchman** [comment]
I wouldn't mind not to be the one who has to update the page(s)  😆 
Was just a suggestion for the case if no one wanted the task ;)

If wanted, I would update one of the 2 pages
